core/storage/feedback/gae_models.py

```diff
@@ -287,3 +287,15 @@ def get_by_exploration_and_thread_id(cls, exploration_id, thread_id):
         Returns None if it doesn't match anything."""
 
         return cls.get_by_id(cls._get_instance_id(exploration_id, thread_id))
+
+
+class FeedbackMessageEmailDataModel(base_models.BaseModel):
```

Comment:
Just a thought: what do you think about the name UnsentFeedbackMessageEmailDataModel? Makes clearer that this is for unsent stuff, not all feedback messages.